Formula/gpg-tui.rb

```diff
@@ -0,0 +1,33 @@
+class GpgTui < Formula
+  desc "Manage your GnuPG keys with ease! 🔐"
+  homepage "https://github.com/orhun/gpg-tui"
+  url "https://github.com/orhun/gpg-tui/archive/refs/tags/v0.1.5.tar.gz"
+  sha256 "eccfaa3089605bb6df2cc49df208a93cc452d8a819b444bf82dfeb90739135ea"
+  license "MIT"
+
+  depends_on "rust" => :build
+  depends_on "gnupg"
+  depends_on "gpgme"
+  depends_on "libgpg-error"
+  depends_on "libxcb"
+
+  def install
+    system "cargo", "install", *std_cargo_args
+
+    completions = share/"completions"
+    mkdir_p completions
+    ENV["OUT_DIR"] = completions
+    system bin/"completions"
+    bash_completion.install completions/"gpg-tui.bash"
+    fish_completion.install completions/"gpg-tui.fish"
+    zsh_completion.install completions/"_gpg-tui"
+
+    rm_rf completions
```

Comment:
```suggestion
    ENV["OUT_DIR"] = buildpath
    system bin/"completions"
    bash_completion.install "gpg-tui.bash"
    fish_completion.install "gpg-tui.fish"
    zsh_completion.install "_gpg-tui"
```
Doesn't this work?

Reply:
Thanks for the suggestion, this works and is more elegant (sorry, this is my first formula).